tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -149,63 +158,101 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         return {
             name: "Pager tooltip tests",
-            tests: {
-                name: "Tooltip visibility",
-                sequence: sequence
-            }
+            tests: [
+                {
+                    name: "Tooltip visibility and contents",
+                    sequence: sequence
+                }
+            ]
         };
     };
 
+    fluid.tests.getTooltipContents = function(listOfData, pageSize) {
+        var tooltipContents = [];
+        for (var i = 0; i < listOfData.length; i += pageSize) {
+            var tooltipContent = [];
+            var pageStartIndex = i;
+            var pageEndIndex = Math.min(pageStartIndex + pageSize -1, listOfData.length - 1);
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageStartIndex].text});
+            tooltipContent.push({nodeName: "b", nodeText: listOfData[pageEndIndex].text});
+            tooltipContents.push(tooltipContent);
+        }
+
+        return tooltipContents;
+    };
+
+    fluid.tests.verifyLinksAndTooltips = function(pager, pageSize, linksList) {
+        fluid.changeElementValue(pager.locate("pageSize"), pageSize);
+        fluid.tests.verifyPagerBar(pager, "top", pageSize);
+        fluid.tests.verifyPagerBar(pager, "top", pageSize);
+        fluid.tests.verifyToolTipsOfPagerBar(pager, "top", pageSize, fluid.tests.getTooltipContents(linksList, pageSize));
+        fluid.tests.verifyToolTipsOfPagerBar(pager, "bottom", pageSize, fluid.tests.getTooltipContents(linksList, pageSize));
+    };
+
+    fluid.tests.getPagerBar = function(pager, location) {
+        return pager[location === "top" ? "pagerBar" : "pagerBar-1"];
+    };
+
+    fluid.tests.verifyPagerBar = function (pager, location, pageSize) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
+
+        //Verify the number of links in the pager bar
+        jqUnit.assertEquals("The number of links in the pager should be according to the page size",
+            expectedNumberOfLinks,
+            fluid.tests.getPagerBar(pager, location).locate("pageLinks").length);
+    };
+
+    fluid.tests.assertVisibleTips = function (pager, message, targetIds, expectedTooltip) {
+        fluid.tests.tooltip.assertVisible(message, pager, targetIds, null, function (tooltip) {
+            jqUnit.assertNode(tooltip.length, expectedTooltip, $("b", tooltip));
+        });
+    };
+
+    fluid.tests.verifyToolTipsOfPagerBar = function (pager, location, pageSize, expectedTooltips) {
+        var expectedNumberOfLinks = Math.ceil(pager.dataModel.length / pageSize);
+
+        for (var x = 0; x < expectedNumberOfLinks; x++) {
+            fluid.tests.verifyPagerToolTip(pager, location, x, expectedTooltips[x]);
+        }
+    };
+
+    fluid.tests.verifyPagerToolTip = function (pager, location, index, expectedTooltip) {
+        var link =  $(fluid.tests.getPagerBar(pager, location).locate("pageLinks").find("a")[index]);
```

Comment:
instead of using `[index]` you can use `.eq(index)` which will give you back a jQuery object. Also there is a lot of chaining here. Maybe separate out the pageBar from the link into separate variables.